### PR TITLE
mtsr_camera_upgrade

### PR DESCRIPTION
**Pull request by rumachan**
Upgrade camera at MTSR. Only install/cameras.csv needs changing as camera already in assets/cameras.csv

Based on following info supplied by RIM
Manufacturer: Mobotix AG
Model: M16
Serial Number: 10.26.194.146
Asset Number: 16044
Location Code: MTSR.01
Sensor Type: camera
Orientation (azimuth): 037T
Dip: 12.5 degrees above horizontal
Height: -2

Camera tilt is 12.5 degrees above horizontal, same as previous install
Camera azimuth is 037 degrees (determined from images not from direct measurement)

Last M15 image at 20220923T01:20:00Z (13:20 on 23/09/2022 local time): https://data.geonet.org.nz/camera/volcano/images/2022/MTSR/MTSR.01/2022.266/2022.266.0120.00.MTSR.01.jpg

First M16 image at 20220923T02:40:00Z (14:40 on 23/09/2022 local time): https://data.geonet.org.nz/camera/volcano/images/2022/MTSR/MTSR.01/2022.266/2022.266.0240.00.MTSR.01.jpg

### CODE REVIEW

File: install/cameras.csv [hunk 1]
```diff
@@ -37,6 +37,7 @@ Mobotix AG,M15,10.18.168.61,WICF,01,0,0,0,0,0,2016-06-01T23:40:00Z,9999-01-01T00
 Mobotix AG,M15,10.20.210.120,KMTP,01,9.5,290,-2.5,0,0,2020-05-07T00:20:00Z,2022-04-12T07:40:01Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
 Mobotix AG,M15,10.20.245.115,TOKR,01,0,0,0,0,0,2018-05-03T00:00:00Z,9999-01-01T00:00:00Z,Karewerewa looking at TeMaari
 Mobotix AG,M15,10.7.46.153,TOKR,01,0,0,0,0,0,2017-10-12T00:40:00Z,2018-05-02T23:55:00Z,
+Mobotix AG,M16,10.26.194.146,MTSR,01,12.5,37,2,0,0,2022-09-23T02:40:00Z,9999-01-01T00:00:00Z,Mangateitei looking at South Ruapehu
```
Comment (staylorofford) [inline install/cameras.csv:40]: The height is said to be -2 in the ticket comment. but is +2 here. I figure this is correct, as generally up is positive with height. Is the ticket comment meaning depth is -2?

Previous install doesn't have 12.5 dip, despite ticket comment saying it did.

Comment (rumachan) [reply install/cameras.csv:40]: Don't know why the ticket says -2 to be honest but I know the camera has to be above the ground surface!

**Comment by rumachan** [comment]
@staylorofford Added 12.5 deg angle to old MTSR entry as per your note. I assume Cam knew it was correct but had never added it. All I knew was it was blank

**Review by staylorofford — APPROVED** [review]
LGTM

File: install/cameras.csv (hunk 1 above)
Comment (Camasher) [reply install/cameras.csv:40]: @rumachan @staylorofford clarifying why I put -2, generally seismometers were given positive values to indicate depth, hence why I chose a negative depth (above ground).

**Pull request merged**